src/Uno.UI/UI/Xaml/UIElement.Pointers.Skia.cs

```diff
@@ -73,17 +76,24 @@ private void CoreWindow_PointerReleased(CoreWindow sender, PointerEventArgs args
 				{
 					foreach(var target in capture.Targets)
 					{
-						var pointerArgs = new PointerRoutedEventArgs(args, pointer, target.Element) { CanBubbleNatively = true };
-						TraverseAncestors(target.Element, pointerArgs, e => e.OnNativePointerUp(pointerArgs));
+						var pointerArgs = new PointerRoutedEventArgs(args, pointer, target.Element);
+						target.Element.OnNativePointerUp(pointerArgs);
 					}
 				}
 				else
 				{
 					PropagateEvent(args, e =>
 					{
-						// Console.WriteLine($"PointerManager.Released [{e}/{e.GetHashCode():X8}");
-						var pointerArgs = new PointerRoutedEventArgs(args, pointer, e) { CanBubbleNatively = true };
-						TraverseAncestors(e, pointerArgs, e2 => e2.OnNativePointerUp(pointerArgs));
+						if (this.Log().IsEnabled(LogLevel.Trace))
+						{
+							this.Log().Trace($"PointerManager.Released [{e}/{e.GetHashCode():X8}");
+						}
+
+						Console.WriteLine($"Pointer released [{e}/{e.GetHashCode():X8}]");
```

Comment:
Codacy found an issue: [Remove this logging statement.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5924508)